src/libstd/sys/unix/ext/process.rs

```diff
@@ -67,10 +67,20 @@ pub trait CommandExt {
     /// an error indicating why the exec (or another part of the setup of the
     /// `Command`) failed.
     ///
+    /// `exec` not returning has the same implications as calling
+    /// [`process::exit`] – no destructors on the current stack or any other
+    /// thread’s stack will be run. Therefore, it is recommended to only call
```

Comment:
No. “Child thread” makes sense in relation to “main thread”, but you can call process::exit on any thread you want.